README.md

```diff
@@ -113,6 +113,19 @@ could render your app at the `/` route with the following view:
 
 Your Ember application will now be served at the `/` route.
 
+## Enabling LiveReload
+In order to get LiveReload correctly up and running for EmberCLI Rails, you can install the [guard](https://github.com/guard/guard) and [guard-livereload](https://github.com/guard/guard-livereload) gems and then run `guard init`and adding the following line to your `Guardfile`
+
+```
+guard 'livereload' do
+  ...
+  watch(%r{(app/{<your-appname>/\w+/(.+\.(js|hbs|html)))})
+  ...
+end
+```
+
+This tells Guard to watch your EmberCLI app directory for any Javascript, Handlebars, or HTML changes and reloading the browser on any file changes.
```

Comment:
Valid point, I'll add that as well. I also realized that I forgot some CSS/SCSS file extensions as well.